docs/client-access-token.md

```diff
@@ -76,7 +76,18 @@ Authorization: Bearer YOUR_ACCESS_TOKEN
 >
 > Make sure to **cache and reuse** the obtained client access token for as long as possible. Do not request a new access token for each API request you make.
 >
-> The response with your access token will include an `expires_in` parameter with the exact expiration time in seconds. You can make your cached token expire after that time has elapsed and request a new one then. Another option is to not look at the expiration time and request a new token as soon as you get a `401` response from an API, indicating that the token has expired.
+> There are two ways to check if your cached token is still valid:
+>
+> 1.  Cache the `expires_in` property included in the token response, and the time that you requested the token. Using these two parameters, you can calculate the expiration time of the token and request a new one when it is expired. Note that if you follow this approach, you should account for clock skew between your server and the APIs' servers, so it's best to already request a new token a couple of minutes before the cached one will expire.
+> 2.  Keep using the same cached token until you get a `401` response from an API endpoint, at which point you can request a new token and perform the failed request again with the new token. Note that you will need to set a maximum number of retries if you follow this approach, to prevent an infinite loop if there happens to be an issue that prevents you from getting a valid token.
+
+<!-- theme: warning -->
+
+> ##### Parsing tokens
+>
+> Do not rely on being able to parse a token as a JWT, for example to check its expiration time. While publiq's access tokens are JWTs right now, this is not guaranteed to always be the case and it might happen that you get a token that is not a parseable JWT at some point in the future.
+>
+> Claims inside the token are also not guaranteed to be stable and may change over time.
```

Comment:
Dit is inderdaad zeer goed om te vermelden. Ik zou zelfs durven zeggen als: clients should not parse the token or rely on any of the claims that might be part of it.

Reply:
Ik dacht dat dat duidelijk was met "Do not rely on being able to parse a token as a JWT, for example to check its expiration time." maar nu ik het herlees kan ik dat beter wat verwoorden om het nog duidelijker te maken. 👍